recipe-server/client/control_new/components/recipes/EditRecipePage.js

```diff
@@ -39,6 +39,14 @@ export default class EditRecipePage extends React.Component {
     recipe: null,
   };
 
+  static ActionMenu = (
```

Comment:
I don't think the form should include any controls that are non-form related. That's the whole point of the details page. I think we should ditch this.